drafts/concise_guide.ipynb

```diff
@@ -161,6 +220,22 @@
     "```"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Enums can be used to constrain the values accepted for a route parameter.\n",
+    "\n",
+    "```python\n",
+    "ModelName = str_enum('ModelName', \"alexnet\", \"resnet\", \"lenet\")\n",
+    "\n",
+    "@rt(\"/models/{nm}\")\n",
+    "def get(nm:ModelName): return nm\n",
+    "\n",
```

Comment:
You can remove this since you're not actually printing it, and the output isn't important here.